packages/terra-status-view/tests/nightwatch/StatusViewNotAuthorized.jsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import Base from 'terra-base';
+import StatusView from '../../lib/StatusView';
+
+const locale = document.getElementsByTagName('html')[0].getAttribute('lang');
+
+export default () =>
+  <Base locale={locale}>
+    <StatusView role="main" id="statusView" variant={StatusView.variants.NOTAUTHORIZED} />
```

Comment:
What's the logic on setting `role="main"` here?

Reply:
It is is allow the tests to pass when checking to make sure they are accessible or not; the rule that was breaking is described here:
https://dequeuniversity.com/rules/axe/3.0/landmark-one-main

I tried turning if off and while that fixed the tests locally; they still failed on Travis CI.  Therefore, I added the role of main to the test samples instead to get the CI build to succeed.

Reply:
Hmm, I have mixed thoughts on this. I lean towards disabling the landmark-one-main accessibility rule for our component tests rather than adding `role="main"` to our component tests to get the linter to pass.

Might try the following:

```
// Before your describe blocks
const ignoredA11yRules = {
  'landmark-one-main': { enabled: false },
};

// In your describe blocks
Terra.should.beAccessible({ viewports, rules: ignoredA11yRules });
```
And then once thats in place we can clear the cache on travis and see if that fixes the issue described of the tests not passing on travis.

Reply:
If clearing the cache on travis causes it to pull in the newest version of axe that has this rule, then all of are accessibility tests in all the other components are going to start failing. We should probably log a separate issue for this.

Reply:
@bjankord What you suggested is how I originally had the code; however, the CI build failed, I will revert the change to use role="main" and try again to see if it passes.

Reply:
Ok I can try clearing the cache on travis too and see if that makes any difference.